pom.xml

```diff
@@ -30,6 +30,10 @@
             <id>jaudiotagger-repository</id>
             <url>https://dl.bintray.com/ijabz/maven</url>
         </repository>
+        <repository>
+            <id>jitpack.io</id>
+            <url>https://jitpack.io</url>
```

Comment:
Can't the cue lib be found in the official maven repo?

Reply:
It couldn't when I made the changes a long time ago, maybe it can now, will check. For now I'm just bringing the changes over to the latest version of airsonic, will look at these issues later.

Reply:
Now using `cuelib-core` from _org.digitalmediaserver_